src/main/java/com/school/project/model/entities/Group.java

```diff
@@ -1,13 +1,33 @@
 package com.school.project.model.entities;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
 import java.util.Date;
 import java.util.List;
 
+@Entity
+@Table(name = "`GROUP`")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 public class Group {
 
+    @Id
+    @Column(name = "ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    @Column(name = "START_DATE", nullable = false)
+    @Temporal(TemporalType.DATE)
     private Date startDate;
+    @OneToOne
+    @JoinColumn(name = "MODULE_ID", nullable = false)
     private Module module;
+    @ManyToOne(cascade = CascadeType.ALL, targetEntity = User.class)
```

Comment:
I think here should be One to many since there won't be situation possible where one user relates to multiple groups as student